modules/nom/src/nom/parsers/json.fz

```diff
@@ -105,7 +105,7 @@ public json =>
         if x.is_empty
           0.0
         else
-          (String.type.from_codepoints x.first)
+          (String.type.from_codepoints x.first.get)
             .parse_u64
             .val 0
             .as_f64 / (10.0*x.count.as_f64))
```

Comment:
Would be nicer to use match instead of `.get`: 
```
        match x.first
          nil        => 0.0
          f Sequence => String.type.from_codepoints f
                              .parse_u64
                              .val 0
                              .as_f64 / (10.0*x.count.as_f64))
```